django/utils/datastructures.py

```diff
@@ -131,17 +131,29 @@ def get(self, key, default=None):
             return default
         return val
 
-    def getlist(self, key, default=None):
+    def _getlist(self, key, default=None, force_list=False):
         """
-        Returns the list of values for the passed key. If key doesn't exist,
-        then a default value is returned.
+        Returns a list of values for the passed key.
+        Used internally to manipulate values list.
+        If force_list is True returns a new copy of values.
         """
         try:
-            return super(MultiValueDict, self).__getitem__(key)
+            values = super(MultiValueDict, self).__getitem__(key)
         except KeyError:
             if default is None:
                 return []
             return default
+        else:
+            if force_list:
+                values = list(values)
+            return values
+
+    def getlist(self, key, default=None):
+        """
+        Returns the list of values for the passed key. If key doesn't exist,
```

Comment:
I just copied style from other comments in same file. Maybe they should be cleaned up at some point?